.github/workflows/main.yml

```diff
@@ -0,0 +1,177 @@
+name: package
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build-package:
+    name: ${{ matrix.config.name }}
+    runs-on: ${{ matrix.config.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+        - {
+            name: "Linux",
+            os: ubuntu-latest,
+            package: 'tar.gz'
+          }
+        - {
+            name: "MacOSX",
+            os: macos-latest,
+            package: 'dmg'
+          }
+        - {
+            name: "Windows",
+            os: windows-latest,
+            package: 'msi'
+          }
+    defaults:
+      run:
+        shell: bash
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+        path: hexrdgui
+
+    - name: Checkout HEXRD master for local build ( PRs only )
+      if: github.ref != 'refs/heads/master'
```

Comment:
By the way, I think these could also be `if: github.event_name == 'pull_request'`. But it's probably not that important.

Reply:
Yup, I think either work in this case